pkg/provision/linode_test.go

```diff
@@ -0,0 +1,139 @@
+package provision
+
+import (
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"github.com/inlets/inletsctl/mock"
+	"github.com/linode/linodego"
+	"net"
+	"testing"
+)
+
+func Test_Linode_Provision(t *testing.T) {
```

Comment:
What is this test testing?

Reply:
It is basically testing all the logic of linode by mocking the external calls to linode.

e.g. It will test if expected options are passed to linode SDK CreateInstance().